.github/workflows/helm-release.yaml

```diff
@@ -77,5 +77,5 @@ jobs:
             chart=${dir##*/}
             echo "Found chart: ${chart}"
             helm package charts-tmp/${chart} --destination dist
-            helm push dist/${chart}-*.tgz oci://ghcr.io/${GITHUB_REPOSITORY}
-          done
\ No newline at end of file
+            helm push dist/${chart}-*.tgz oci://ghcr.io/${GITHUB_REPOSITORY%/*}/charts
```

Comment:
In `${GITHUB_REPOSITORY%/*}` what is the `%/*` doing ?

Reply:
https://www.cazzulino.com/github-actions-repository.html

Reply:
Why not use `${{ github.repository_owner }}` ?

Reply:
that makes sense yeah !